lib/paper_trail/version_concern.rb

```diff
@@ -379,16 +379,22 @@ def sibling_versions
     # The version limit can be global or per-model.
     #
     # @api private
-    #
-    # TODO: Duplication: similar `constantize` in Reifier#version_reification_class
     def version_limit
-      if self.class.item_subtype_column_present?
-        klass = (item_subtype || item_type).constantize
-        if klass&.paper_trail_options&.key?(:limit)
-          return klass.paper_trail_options[:limit]
-        end
+      if limit_option?(item.class)
+        item.class.paper_trail_options[:limit]
+      elsif base_class_limit_option?(item.class)
+        item.class.base_class.paper_trail_options[:limit]
```

Comment:
This is cool, I had forgotten about `base_class`.

It seems this branch is untested, according to SimpleCov.

Reply:
Yes, I'll get something added.